Samurai/Modules/ConversionModule.cs

```diff
@@ -1,15 +1,12 @@
-using System;
 using System.Linq;
 using System.Threading.Tasks;
+using AcidChicken.Samurai.Assets;
+using AcidChicken.Samurai.Components;
```

Comment:
All of the local namespace reference instructions have to be included in the local namespace block.